src/main/scala/mastermind/Mastermind.scala

```diff
@@ -12,20 +12,38 @@ type Board = String
 val validColors = List('B', 'Y', 'R', 'G')
 
 /** Get a random color from the list of valid colors */
-def getRandomColor(): Color =
-  ???
+def getRandomColor(): Color = {
+  val rand = scala.util.Random()
+  val colorIndex = rand.nextInt(4)
+  validColors(colorIndex)
+}
```

Comment:
I like how you split methods into different lines so we have readable chunks instead of one long chained expression